docs/arch.md

```diff
@@ -5,23 +5,47 @@ The main goal of the Insights Operator is to periodically gather anonymized data
+- `reportEndpoint` - download endpoint. From this endpoint, the Insights operator downloads the latest Insights analysis. Overwritten by `dataReporting/downloadEndpoint` from the configmap. Default value is `https://console.redhat.com/api/insights-results-aggregator/v2/cluster/%s/reports` (where `%s` must be replaced with the cluster ID).
 - `reportPullingDelay` - the delay between data upload and download. Default value is `60s`
 - `reportPullingTimeout` - timeout for the Insights download request.
 - `reportMinRetryTime` - the time after which the request is retried. Default value is `30s`
-- `scaEndpoint` - the endpoing for downloading the Simple Content Access(SCA) entitlements. Default value is `https://api.openshift.com/api/accounts_mgmt/v1/certificates`
-- `scaInterval` - frequency of the SCA entitlements download. Default value is `8h`.
-- `scaPullDisabled` - flag to disable the SCA entitlements download. Default value is `false`.
-- `clusterTransferEndpoint` - the endpoint for checking & download cluster transfer data (updated `pull-secret` data). Default values is `https://api.openshift.com/api/accounts_mgmt/v1/cluster_transfers/`
-- `clusterTransferInterval`  - frequency of checking available cluster transfers. Default value is `24h`.
-- `conditionalGathererEndpoint` - the endpoing providing conditional gathering rules definitions. Default value is `https://console.redhat.com/api/gathering/gathering_rules`.
-
+- `scaEndpoint` - the endpoing for downloading the Simple Content Access(SCA) entitlements. Default value is `https://api.openshift.com/api/accounts_mgmt/v1/certificates`. Overwritten by `sca/endpoint` from the configmap.
+- `scaInterval` - frequency of the SCA entitlements download. Overwritten by `sca/interval` from the configmap. Default value is `8h`.
+- `scaPullDisabled` - flag to disable the SCA entitlements download. Overwritten by `sca/disabled` from the configmap. Default value is `false`.
+- `clusterTransferEndpoint` - the endpoint for checking & download cluster transfer data (updated `pull-secret` data). Overwritten by `clusterTransfer/endpoint` from the configmap. Default values is `https://api.openshift.com/api/accounts_mgmt/v1/cluster_transfers/`
+- `clusterTransferInterval`  - frequency of checking available cluster transfers. Overwritten by `clusterTransfer/interval` from the configmap. Default value is `24h`.
+- `conditionalGathererEndpoint` - the endpoing providing conditional gathering rules definitions. Overwritten by `dataReporting/conditionalGathererEndpoint` from the configmap. Default value is `https://console.redhat.com/api/gathering/gathering_rules`.
+- `disableInsightsAlerts` - disables the insights alerts for when the insights operator is disabled. Overwritten by `alerting/disabled` from the configmap. Default value is `false`.
```

Comment:
> disables the insights alerts for when the insights operator is disabled

I think I don't understand this sentence. This disables all the alerts registered by the Insights Operator by default.

Reply:
Corrected, thanks!